bin/install-app.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+# This program and the accompanying materials are
+# made available under the terms of the Eclipse Public License v2.0 which accompanies
+# this distribution, and is available at https://www.eclipse.org/legal/epl-v20.html
+# 
+# SPDX-License-Identifier: EPL-2.0
+# 
+# Copyright Contributors to the Zowe Project.
+if [ $# -eq 0 ]
+    then
+    echo "Usage: $0 AppPath"
+    exit 1
+fi
+utils_path=$(pwd)/../../zlux-server-framework/utils
+if [ "$1" = "/*" ]
```

Comment:
Could you please explain what this means?

Reply:
I believe this was to detect relative versus absolute paths.

Reply:
Well, I don't think it works that way. Relative path is always relative to $(pwd) anyway, so no need to do that unless you later cd